frontend/pages/admin/charities/index.tsx

```diff
@@ -27,7 +65,32 @@ const AdminCharities = () => {
             Create
           </Button>
         </Stack>
+
+        <SimpleTable
+          rows={charities ?? []}
+          isLoading={!charities}
+          columns={[
+            { title: 'ID', key: 'id' },
+            { title: 'Logo', key: 'logoBase64', transformValue: createLogos },
```

Comment:
Will we display the description and image of the charity anywhere? Since we don't have a show page, it needs to be displayed in the list. So this is an exception but list charities should actually fetch `CharityData[]` and not `CharityListData[]`. The type can be renamed for clarity if necessary.

Reply:
It would be quite cluttered in the table IMO, and could take 30 seconds if we had to list all CharityData[]. I do not think we should have an endpoint that lists all data like that. 

For now, the only way to access a charity's data would be by going to their edit page. I can make a simple `/charities/{id}/view` page instead in a new PR.

Reply:
Okay I think that works, we can make the view page low priority and do it later if we have time